app/components/Reactions/Reaction.tsx

```diff
@@ -91,28 +91,34 @@ const Reaction = ({
     return <></>;
   }
 
-  let tooltipContent =
-    showPeople && optimisticUsers?.map((user) => user.fullName).join(', ');
-  tooltipContent = tooltipContent ? `${tooltipContent} reagerte med ` : '';
-  tooltipContent += emoji;
+  const tooltipContentNames =
+    showPeople &&
+    optimisticUsers
+      ?.map((user) => user.fullName)
+      .sort()
+      .map((name) => <div key={name}>{name}</div>);
+  const tooltipContent = tooltipContentNames ? (
+    <div>
+      {tooltipContentNames}
+      <p>har reagert med {emoji}</p>
```

Comment:
```suggestion
     	 har reagert med {emoji}
```

The `<p>` adds a bottom margin - I think it looks better without